protoc-gen-openapiv2/internal/genopenapi/testdata/generator/path_item_object.swagger.yaml

```diff
@@ -22,6 +22,75 @@ paths:
           in: query
           required: false
           type: string
+        - name: uuid
+          in: query
+          required: false
+          type: string
+        - name: id
+          in: query
+          required: false
+          type: string
+      tags:
+        - YourService
+  /api/echo/{id}:
+    get:
+      operationId: YourService_Echo4
+      responses:
+        "200":
+          description: A successful response.
+          schema:
+            $ref: '#/definitions/StringMessage'
+      parameters:
+        - name: id
+          in: path
+          required: true
+          type: string
+        - name: value
+          in: query
+          required: false
+          type: string
+      tags:
+        - YourService
+  /api/echo/{uuid}:
+    get:
+      operationId: YourService_Echo3
+      responses:
+        "200":
+          description: A successful response.
+          schema:
+            $ref: '#/definitions/StringMessage'
+      parameters:
+        - name: uuid
```

Comment:
without this fix `Id` would also come up here, it's not now because `uuid` is of type `one_of`